content/install_maintain/agents/installation.md

```diff
@@ -130,24 +130,30 @@ every agent.
    node instance, or form the deployment logs:    
    ![Script in logs]( /images/manager/agent_installation/logs_script.webp )
 1. Login to the agent's VM.
+1. Prepare the agent directory
+   ```shell
+   sudo mkdir -p /opt/cloudify-agent-6.3.0-ga/
+   sudo chown $(whoami) /opt/cloudify-agent-6.3.0-ga/
+   mkdir -p /opt/cloudify-agent-6.3.0-ga/container_nwxqiu/cloudify/ssl
```

Comment:
Anywhere you're using the agent's identified (container_nwxqiu), you should probably be using a stirng that is noted at the beginning?